netmanager-app/app/(authenticated)/sites/create-site-form.tsx

```
@@ -34,11 +34,32 @@ import { useQueryClient } from "@tanstack/react-query";
+import { useApproximateCoordinates } from "@/core/hooks/useSites";
+import { AxiosError } from "axios";
+import Error from "next/error";
 
 const siteFormSchema = z.object({
   name: z.string().min(2, {
     message: "Site name must be at least 2 characters.",
   }),
+  latitude: z.string().refine(
+    (val) => {
+      const num = parseFloat(val);
+      return !isNaN(num) && num >= -90 && num <= 90;
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)